api/python/cellxgene_census/src/cellxgene_census/py.typed

```diff
@@ -0,0 +1 @@
+# Marker file for PEP 561.  The cellxgene_census package uses inline types.
```

Comment:
```suggestion
# Marker file for PEP 561. The cellxgene_census package uses inline types.
```

Reply:
Lolz, will fix.  This was a literal cut & paste from the mypy marker file.